pom.xml

```diff
@@ -1487,5 +1488,19 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>release-profile</id>
```

Comment:
shall we add it here too @Xenorith  `-Prelease-profile`
https://github.com/Alluxio/alluxio/blob/384f944fa6c6986adeca8e598b62feebd7872276/dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go#L99

Reply:
from my understanding but correct me if i'm wrong, `-Prelease-profile` is only necessary when building artifacts to be published to maven. in all other cases, we would not want this profile to be enabled

Reply:
@LuQQiu can you confirm that we do not want to append this profile argument to `generate-tarball.go` because this doesn't get executed when publishing to maven?

Reply:
Not quite sure if we need it when generating the tarball. But publish to maven doesn't have relationships with generate-tarball.go

Reply:
@LuQQiu Thanks for your reply, as publish to maven doesn't have relationships with generate-tarball.go, so I think we don't need to add `-Prelease` here.